front/src/modules/object-record/object-filter-dropdown/components/ObjectFilterDropdownRecordSelect.tsx

```diff
@@ -31,6 +31,10 @@ export const ObjectFilterDropdownRecordSelect = () => {
     recordToSelect: SelectableRecord,
     newSelectedValue: boolean,
   ) => {
+    if (loading) {
+      return;
```

Comment:
won't this generate a glitch?

Reply:
It can, one way to handle it would be to make the selectable menu items disabled while the request is loading.